### DISCUSSION TITLE
a black box appears on the face when merging with the original image

### DISCUSSION THREAD

**Discussion by mounterx** [Bing-su/adetailer#130] [Q&A]
After repairing the face, a black box appears on the face when merging with the original image，
修复面部后，与原始图像合并时，面部会出现一个黑框，

**Comment by DivanoDova** [comment]
This also happened to me today 2 of my output came with a black square

**Comment by CycloneRing** [comment]
Same, Did you find solution?

**Comment by okay456okay** [comment]
add  --no-half to command arguments, like:
./webui.sh --no-half
My computer is macbook pro m2. 
This works for me.
